src/models/thread.ts

```diff
@@ -310,6 +313,11 @@ export class Thread extends ReadReceipt<ThreadEmittedEvents, ThreadEventHandlerM
         const lastReply = this.lastReply();
         const isNewestReply = !lastReply || event.localTimestamp >= lastReply!.localTimestamp;
 
+        if (isNewestReply) {
```

Comment:
workaround for not refetching the thread root (on refetch, we'd get the latest reply event ID in the event's content)